source/github-helpers/index.ts

```diff
@@ -160,7 +160,11 @@ export function triggerConversationUpdate(): void {
 
 // Fix z-index issue https://github.com/refined-github/refined-github/pull/7430
 export function fixFileHeaderOverlap(child: Element): void {
-	child.closest('.container')!.classList.add('rgh-z-index-5');
```

Comment:
You can just turn the exclamation mark into ?.

Reply:
Oh, I forget its ts file. hhhhh